src/ucp/core/ucp_ep.c

```diff
@@ -3143,6 +3143,11 @@ static void ucp_ep_config_print(FILE *stream, ucp_worker_h worker,
     }
     fprintf(stream, "#\n");
 
+
+    if (worker->context->config.ext.proto_enable) {
+        return;
+    }
```

Comment:
Please move this block before the line `fprintf(stream, "#\n");` to remove extra empty line in the output of `ucx_info -u t -e`.
We adding an empty line in `ucp_ep_print_info_internal` after the `ucp_ep_config_print` call.

Reply:
fixed and force-pushed due to small pr